### PR TITLE
feat: Add `await` option to both api and cli

### PR DESCRIPTION
**Pull request by daedal-knickerbockers**
- When `--await` / `await` option is set, templates with `await` will validate

Fixes https://github.com/RyanZim/EJS-Lint/issues/81

### CODE REVIEW

**Comment by daedal-knickerbockers** [comment]
Hm, I can't seem to get the cli tests run under windows which makes debugging the current issue difficult. Will fix this once I get the tests running.

**Comment by daedal-knickerbockers** [comment]
@RyanZim Seems to be all good now - could you check if it's okay this way?

File: index.js [hunk 1]
```diff
@@ -45,7 +45,10 @@ module.exports = function lint(text, opts = {}) {
       }
     })
     .join('');
-  return check(js);
+  const checkOptions = {
+    allowAwaitOutsideFunction: opts.await || false,
```
Comment (RyanZim) [inline index.js:49]: Can just cast to boolean here.
```suggestion
    allowAwaitOutsideFunction: !!opts.await,
```

**Review by RyanZim — CHANGES_REQUESTED** [review]
One tiny nitpick; other than that, LGTM.

**Comment by daedal-knickerbockers** [comment]
Sure, change is commited :-)

**Review by RyanZim — APPROVED** [review]
LGTM, thanks for including tests!

**Pull request merged**